contracts/Governance/Timelock.sol

```diff
@@ -34,8 +34,8 @@ contract Timelock {
     );
 
     uint public constant GRACE_PERIOD = 14 days;
-    uint public constant MINIMUM_DELAY = 2 days;
-    uint public constant MAXIMUM_DELAY = 30 days;
+    uint public constant MINIMUM_DELAY = 800; // About 1 hour, 3 secs per block (20 * 60)
+    uint public constant MAXIMUM_DELAY = 28800; // About 24 hours, 3 secs per block (20 * 60 * 24)
```

Comment:
But the delay is in seconds here

Reply:
I propose we keep the upper bound > 2 days because our current timelock has a delay of 2 days

Reply:
I'm 100% sure it's in seconds: https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/Governance/Timelock.sol#L89